src/browser/api/window.js

```diff
@@ -1558,24 +1586,81 @@ Window.removeEventListener = function(identity, type, listener) {
     ofEvents.removeListener(route.window(type, browserWindow.id), listener);
 };
 
+function areNewBoundsWithinConstraints(options, width, height) {
+    const {
+        minWidth,
+        minHeight,
+        maxWidth,
+        maxHeight,
+        aspectRatio
+    } = options;
+
+    if (width === undefined && height === undefined) {
+        return true;
+    }
 
-Window.resizeBy = function(identity, deltaWidth, deltaHeight, anchor) {
+    if (height === undefined) {
+        return (width >= minWidth) && (maxWidth === -1 || width <= maxWidth);
```

Comment:
how about maxWidth\Height <=0 here? I'm guessing users who want to remove max/min limitation would set it's value to 0.

Reply:
Anybody got info on what happens when you set maxWidth/Height to 0 vs -1? Seems like most of the code is expecting -1.

Reply:
https://appoji.jira.com/browse/RUN-5019

Reply:
With that in mind, should I leave as is then?

Reply:
as I see it - some users will try passing falsy values in an attempt to cancel a min/max, let's save them the trip to the docs.